Add json output option (#8)
im-open/yaml-file-validation: README.md

## Inputs

| Parameter          | Is Required | Description                                                                                                                  |
| ------------------ | ----------- | ---------------------------------------------------------------------------------------------------------------------------- |
| `yaml-file-path`   | true        | The path of the yaml file to validate.                                                                                       |
| `schema-file-path` | false       | The schema file used to validate yaml file.  If omitted or set to "SAM", the IM-OPEN SAM schema format will be used.         |
| `log-level`        | false       | The severity level of information to include the action's logging. Accepted values are information, warning, and failure.    |

> | `output-json`      | false       | Specify whether to oupt a json version is output accepts true or false. It will only produce an output if no failures occur. |

Reviewer: There is a typo and some awkward wording here. Here's my stab at correcting it, but feel free to ignore my suggestion and do something better since I'm not totally sure what this outputted json is for. In fact, it might make sense to add some sort of explanation of what the json document is/will look like. Maybe not here but in the `Outputs` section? Or maybe it makes sense here. I'll leave it up to you!

```suggestion
| `output-json`      | false       | Specify whether or not to output a json document with the results. Accepts true or false. It will only produce an output if no failures occur. |
```
